cli/cmd/inject_test.go

```diff
@@ -48,21 +48,21 @@ func testUninjectAndInject(t *testing.T, tc testCase) {
 }
 
 func TestUninjectAndInject(t *testing.T) {
-	defaultConfig := newConfig()
-	defaultConfig.global.Version = "testinjectversion"
+	_, defaultConfig, _ := testInstallOptions().validateAndBuild()
```

Comment:
Any reasons why skipping the error check here and in the next two calls? The tests still seem to pass with the checks.

Reply:
we check this in another place so it seemed redundant to check everywhere, but i'll create a wrapper that does it one place.